decidim-conferences/app/controllers/decidim/conferences/conference_widgets_controller.rb

```diff
@@ -8,7 +8,7 @@ class ConferenceWidgetsController < Decidim::WidgetsController
       private
 
       def model
-        @model ||= Conference.find_by(slug: params[:conference_slug])
+        @model ||= Conference.published.find_by(slug: params[:conference_slug])
```

Comment:
Here we may have a possible security issue when multi tenancy is involved. Reading the piece of code, it seems that is possible to access the widgets cross organization.

Reply:
Solved with 059fd6067a